i18n/en/docusaurus-plugin-content-docs/current/tariffs.mdx

```diff
@@ -140,6 +140,46 @@ tariffs:
 
 In this example, there is a surcharge of 10 ct/kWh.
 No surcharge is calculated from 10:00 PM to 6:00 AM.
+
+### Exchange Prices with Season and Time Component
+Useful for "Tafiff model 3" where tariff is different between summer and winter and furthermore tariff changes during the day.
```

Comment:
Can you rephrase this in a more general ways? I assume tariff model 3 ist DK specific?